<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>specDescriptionWindow</class>
 <widget class="QMainWindow" name="specDescriptionWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>149</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>374</width>
    <height>149</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>374</width>
    <height>149</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>کپشن  - ویدئو ویژه</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/edit.png</normaloff>../icons/edit.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="mainGroupBox">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>6</y>
      <width>361</width>
      <height>138</height>
     </rect>
    </property>
    <property name="title">
     <string>توضیح ویدئو</string>
    </property>
    <widget class="QPushButton" name="cancelPushButton">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>103</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>انصراف</string>
     </property>
    </widget>
    <widget class="QPushButton" name="savePushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>103</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ذخیره و خروج</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="descriptionTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
